src/engraving/compat/midi/compatmidirenderinternal.cpp

```diff
@@ -424,7 +425,7 @@ static void renderSnd(EventsHolder& events, const Chord* chord, int noteChannel,
 
         for (int t = change.first.ticks(); t <= endPoint; t++) {
             int mult = multEvents.val(Fraction::fromTicks(t));
-            if (mult == lastVal || mult == CONVERSION_FACTOR) {
+            if (mult == lastVal || muse::RealIsEqual(mult, CONVERSION_FACTOR)) {
```

Comment:
This one doesn't make sense because `CONVERSION_FACTOR` comes from an `int` too
(It is questionable whether `CONVERSION_FACTOR` should exist at all)

Reply:
I'll make it an `int` and cast to `double` where needed (i.e. at the division a few lines down)